web/src/main/java/org/openmrs/web/WebUtil.java

```diff
@@ -420,23 +418,19 @@ public boolean supportsPropertyName(String propertyName) {
 		return OpenmrsConstants.GP_SEARCH_DATE_DISPLAY_FORMAT.equals(propertyName);
 	}
 	
-	public static void setDefaultDateCache(String defaultDateCache) {
-		WebUtil.defaultDateCache = defaultDateCache;
-	}
-	
 	/**
 	 * @see org.openmrs.api.GlobalPropertyListener#globalPropertyChanged(org.openmrs.GlobalProperty)
 	 */
 	@Override
 	public void globalPropertyChanged(GlobalProperty newValue) {
-		setDefaultDateCache(null);
+		throw new UnsupportedOperationException();
```

Comment:
Why throw an exception?

Reply:
Well since the property `defaultDateCache` was removed prior to this issue requirements, this method was just initially nullifying that property. So this Operation is no longer supported.

Reply:
So since the property was removed, should I just put a comment place holder like

`// Do nothing`

Reply:
@dkayiwa  I see it makes no sense.

Since we removed the defaultDateCache, does it still make sense of this
class being a GlobalPropertyListener? according to my understanding, this
property should either remain could be it's used somewhere behind the
curtains, or this class ceases being a GlobalPropertyListener.

How do you see?

Reply:
Other things can still be done in the globalPropertyChanged call back. Therefore it has nothing to do with defaultDateCache

Reply:
Well I now know that globalPropertyChanged call back has nothing to do with defaultDateCache. Here I ask, why is just setting defaultDateCache to null yet it has nothing to do with it? 
And which other thing can be done with the callback?
And to which globalProp is this callback made?